packages/ui/Scrollable/ScrollableNative.js

```diff
@@ -841,8 +846,10 @@ class ScrollableBaseNative extends Component {
 			this.props.clearAllOverscrollEffects();
 		}
 
-		this.setOverscrollStatus('horizontal', overscrollTypes.none);
-		this.setOverscrollStatus('vertical', overscrollTypes.none);
+		this.setOverscrollStatus('horizontal', 'before', overscrollTypes.none);
+		this.setOverscrollStatus('horizontal', 'after', overscrollTypes.none);
+		this.setOverscrollStatus('vertical', 'before', overscrollTypes.none);
+		this.setOverscrollStatus('vertical', 'after', overscrollTypes.none);
```

Comment:
The 4th parameter for `ratio` was omitted. Is it OK or is it better to add it for this case?

Reply:
It does not make a trouble for now, but it's a bug actually. Thank you for pointing out.